wger/manager/views/schedule.py

```diff
@@ -84,6 +88,8 @@ def view(request, pk):
 
     template_data['uid'] = uid
     template_data['token'] = token
+    template_data['is_owner'] = is_owner
+    template_data['owner_user'] = user
 
```

Comment:
A little thing, you need to add ``template_data['show_shariff'] = is_owner`` so the sharing buttons are rendered in the template